sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -14,9 +14,10 @@
 - Renamed `GetSharePropertiesResult` to `ShareProperties`.
 - Renamed `GetShareStatisticsResult` to `ShareStatistics`.
 - Renamed `GetShareDirectoryPropertiesResult` to `ShareDirectoryProperties`.
-- Renamed `GetShareFilePropertiesResult` to `ShareFileServiceProperties`.
+- Renamed `GetShareFilePropertiesResult` to `FileServiceProperties`.
+- Changed the return type of `ShareClient::GetAccessPolicy` to `ShareAccessPolicy`.
```

Comment:
Fix the source as well to match this change:
https://github.com/Azure/azure-sdk-for-cpp/blob/a5bc8fe0efa4534a45692c2787642ba1f073a6da/sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_service_client.hpp#L89-L102